src/main/java/de/tum/in/www1/artemis/web/rest/dto/AssessmentUpdateDTO.java

```diff
@@ -0,0 +1,29 @@
+package de.tum.in.www1.artemis.web.rest.dto;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import jakarta.annotation.Nullable;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+import de.tum.in.www1.artemis.domain.ComplaintResponse;
+import de.tum.in.www1.artemis.domain.Feedback;
+
+/**
+ * @param feedbacks         the updated feedback list
+ * @param complaintResponse the corresponding complaint response
+ * @param assessmentNote    the assessment note
+ */
+@JsonInclude(JsonInclude.Include.NON_EMPTY)
+public record AssessmentUpdateDTO(List<Feedback> feedbacks, ComplaintResponse complaintResponse, @Nullable String assessmentNote) implements AssessmentUpdate {
+
+    /*
+     * Checks if the feedbacks list is null. If it is, it assigns it a default value of an empty list.
+     */
+    public AssessmentUpdateDTO {
```

Comment:
Again, we should check if we can just adjust the surrounding code to support null values.

Reply:
Apparently, I misunderstood the expected behavior. The server correctly throws a BadRequestAlertException if the list is null and the test was wrong. I now fixed the test.